teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -152,9 +198,114 @@ With regards to this checklist, reviewers should do the following:
+
+\*NOTE: Gems will be explained in the next step so don’t worry about it yet.
+
+* [ ] **Gems**
+
+  Gems are something that students will receive upon completing a set of activities. Due to the difference in difficulty for each set of activities across all the modules, a set amount of gems does not make sense. To solve this issue, one of our members had created a Python script that you can run to calculate the number of gems that you should record in the readme of each activity.
+
+The idea is that if you per-assign a certain number of gems to each hard card and you know the amount of mediums and easy's under that given hard card, you can use the algorithm to calculate the entirety of the gem amounts for the rest of the cards.
+
+The script is written to assign points based on the types of cards. It’s in Python so make sure you have some sort of Python editor available for use. Download and open to follow along:  
+Calculates the amount of exp to subtract from medium
+
+**and easy cards for a single hard card \(NOT the entire lab\)**
+
+def assignPoints\(basePts, numCards\):
+
+```text
+  # subtract the bonus
+  bonusRate = 0.1
+  bonus = bonusRate * basePts
+  basePtsNoBonus = basePts - bonus
+
+  # get amount to subtract for medium and easy cards
+  totalWeights = numCards["medium"] + numCards["easy"] * 2
+  subAmountM = basePtsNoBonus / totalWeights
+  subAmountE = subAmountM * 2
+
+  # floor amts
+  subAmountM //= 1
+  subAmountE //= 1
+  print("Hard card: " + str(basePts))
+  print("Medium card: " + str(subAmountM))
+  print("Easy card: " + str(subAmountE))
+
+  def main(): 
+      numMed = int(input("Enter number of medium cards under this hard card: ")) 
+      numEasy = int(input("Enter number of easy cards under this hard card: ")) 
+      basePts = int(input("Enter the exp split amount for this hard card: "))
+
+      numCards = {"medium" : numMed, "easy" : numEasy}
+      assignPoints(basePts, numCards)
+
+  n = 0 
+  while (n >= 0): 
+      main() 
+      n -= 1
+```
+
+_Step 1:_ Open the file with a Python editor.
+
+_Step 2:_ If you only need it once, just run the program. If you want to calculate more than once, you can change the number that the variable N \(line 30\) is assigned as. It’s set up to loop N times.
+
+```text
+                     ![](https://lh6.googleusercontent.com/MX-Beqd-BXYmCmBiTk6uFs-ZCIFyXAFP_xLaktc8kyKe3_VOtJnVHYnS8j7b917p8jns9ihDTFZJmhgXO4mQNZD0AGWtdri8dYQkxa_nRoAkMh8NvBTJdMJgFPwbfYNv3_-clbn0)
+```
+
+_Step 3:_ Enter all the inputs requested and press enter.
+
+```text
+                   ![](https://lh3.googleusercontent.com/akLNN7ownnjpvsJm1ZmDl-gpzdHHU7WIis6DJ8RYbG0oRrmdwrm-ea-jlzJXVJ84MU3CqlUK0BnZVBEBbTrnZMzMBytgfNKGrrwA8HLg2wDEnJgKWQRYGk4iby8MT7gA3lLdxkqO)
+```
+
+_Step 4:_ With the results, please consult Kevin for the number of gems.
```

Comment:
should be emphasized that they should consult with me on how many gems to give each hard card. everything else is given.